udata_gouvfr/harvesters/maaf.py

```diff
@@ -168,15 +164,15 @@ def process(self, item):
         if metadata.get('license_id'):
             dataset.license = License.objects.get(id=metadata['license_id'])
 
-        if (metadata.get('temporal_coverage_from')
-                and metadata.get('temporal_coverage_to')):
+        if (metadata.get('temporal_coverage_from') and
```

Comment:
This is W503 from http://pep8.readthedocs.org/en/latest/intro.html
(I was surprised too)